mne/viz/evoked.py

```diff
@@ -803,7 +803,7 @@ def plot_evoked_topo(evoked, layout=None, layout_scale=0.945,
         determines the upper and lower subplot limits. e.g.
         ylim = dict(eeg=[-20, 20]). Valid keys are eeg, mag, grad, misc.
         If None, the ylim parameter for each channel is determined by
-        the maximum absolute peak.
+        the minimum and maximum peak.
```

Comment:
Does this mean that each channel has a different set of ylims? If so, could that be made explicit in the doc? Something like, "such that each channel in the figure will have a different set of y limits". As I think most peoples intuition when looking at a plot is that all lines are comparable.

Or maybe I misunderstood the doc string.

Reply:
Each channel type does have its own set of ylims and yeah, I should be made explicit in the docstring.

Reply:
Currently, the only indication of that in the docstring is the "for each channel" bit.

Reply:
Editing the docstring in https://github.com/mne-tools/mne-python/pull/9285/commits/41b8aea0395090f5772a34a354d1f75408dfc530 to make it clearer.

Reply:
Got it, thanks.